NMnode/chains.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "NMnode",
+  "chains": [
+    {
+      "name": "sifchain",
+      "address": "sifvaloper1ujkquuvvpdyyq8qk0xn6j8y98uel5rf6nwvukr",
+      "restake": {
+        "address": "sif1k6gc7fs8m45vadweda593jrt86stlhjcdheur2",
+        "run_time": "Every 24 hours | 21:00 UTC+2",
```

Comment:
This would need to just be `19:00` as the value is processed by the UI. Time is in UTC. A single time implies 24h.

You can also specify multiple times with an array, or an interval string with 'every 15 minutes' etc